pkgs/top-level/all-packages.nix

```diff
@@ -1247,6 +1247,7 @@ with pkgs;
   aptly = callPackage ../tools/misc/aptly { };
 
   ArchiSteamFarm = callPackage ../applications/misc/ArchiSteamFarm { };
+  ASF-ui = callPackage ../applications/misc/ArchiSteamFarm/web-ui { };
```

Comment:
I think this should be under ArchiSteamFarm.ui because realistically no one would want to use it without ASF or the service.

Reply:
How can I move an entire derivation under an output from a different one?

Reply:
```nix
outputs = [ "out" "ui" ];

postInstall = ''
  moveToOutput $out/dist $ui
'';

Reply:
I've pushed [code that should work](https://github.com/NixOS/nixpkgs/blob/fa97f032d4ac9be0aab78c481b83117d6eca01ed/pkgs/applications/misc/ArchiSteamFarm/default.nix#L46), but it keeps popping an error with this:
```
/nix/store/06nwjwvv21bp884l3nhm3dyyj01gwizx-archisteamfarm-5.2.0.10.drv' failed to produce output path for output 'ui' at '/nix/store/06nwjwvv21bp884l3nhm3dyyj01gwizx-archisteamfarm-5.2.0.10.drv.chroot/nix/store/vapfgr38zq9qyq9qygcaj0wasihmyn2y-archisteamfarm-5.2.0.10-ui
```

Reply:
ahh, if it is a totally separate package then passthru should be used.

```nix
passthru.ui = callPackage ./web-ui { }
```